src/index.html

```diff
@@ -1,14 +1,67 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <!-- prettier-ignore -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" >
+    <!-- prettier-ignore -->
+    <link rel="preconnect" href="https://fonts.gstatic.com"  >
+    <!-- prettier-ignore -->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    >
+    <!-- prettier-ignore -->
+    <meta charset="UTF-8" >
+    <!-- prettier-ignore -->
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <!-- prettier-ignore -->
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
+    <!-- prettier-ignore -->
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <!-- prettier-ignore -->
+    <link rel="stylesheet" href="./style.css" >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
```

Comment:
you should use classes to style your elements, header should have class and you can use it to in styles.css